deleteAndLoadSnapsets.sh

```diff
@@ -2,6 +2,7 @@
 
 STATIC_DATA_DIR=${STATIC_DATA_DIR:-/home/node/universal/testData/dbData}
 BUILD_DATA_DIR=${BUILD_DATA_DIR:-/home/node/universal/build/dbData}
+NODE_PATH=${NODE_PATH:-/home/node/universal}
```

Comment:
I am thinking if a var name like `UNIVERSAL_PATH` is better than `NODE_PATH` because this path is defined for accessing scripts/ in the universal repo. What do you think?

Reply:
There is a way to eliminate the `NODE_PATH` variable entirely.  Related:  I discovered that since the `deleteSnapSets.js` script uses infusion, an `npm install infusion` is required here.  Simply cloning universal does not provide an instance of infusion within its `node_modules` folder.  If infusion is so installed, and a `cd universal` is executed after it is cloned (already done), that's enough for the `node` command to find infusion.  No need for `NODE_PATH`.